.github/workflows/automerge.yml

```diff
@@ -0,0 +1,37 @@
+name: Automerge
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  # replace "github_username" with your GitHub username
+  # replace "github.com/username/repo.git" with your GitHub repo path
+  # do NOT replace ${{secrets.GITHUB_TOKEN}}, GitHub will take care of it
+  MY_REPO: https://GeospatialResearch:${{secrets.GITHUB_TOKEN}}@github.com/GeospatialResearch/Digital-Twins-191.git
+
+  # replace "long-lived_branch_name" with your branch name
+  LONG_LIVED_BRANCH: master
+
+  # replace it with the path to master repo
+  MASTER_REPO: https://github.com/GeospatialResearch/Digital-Twins-191.git
+
+  # replace "master" with your master branch name
+  DEFAULT_BRANCH: otakaro_digital_twin
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Merge with master
+      run: |
+        git clone ${{env.MY_REPO}} -b ${{env.DEFAULT_BRANCH}} tmp
```

Comment:
I think it would be good to use checkout actions instead, see how its done in `Build Documentation.yml`

Reply:
This might not be fully neccessary, but it would allow the commits to come from GRI-Admin and make things a bit more consistent with other code. It might be too much work though so evaluate it and let me know

Reply:
I have changed it and have seen no difference, so I think it should be fine. However, you might want to check again if I did anything wrong.